lib/common.js

```diff
@@ -526,6 +525,21 @@ function html(markdown,header,options) {
     return preface+md.render(markdown);
 }
 
+function loadMD(sourceSection, destSection, mdFieldPrefix) {
+    for (let field in sourceSection) {
+        if (field.startsWith(mdFieldPrefix)) {
+            //Replacing '-' so that field can be used in dot templates
+            let newFieldName = field.split("-").join("_");
+            try {
+                destSection[newFieldName] = md.render(fs.readFileSync(sourceSection[field],'utf8'));
+            }
+            catch (ex) {
+                throw ex;
```

Comment:
Is there a reason to `catch` and then `throw` the same exception? If we're `catch`ing it, doing a `console.error(ex.message)` is probably sufficient.